packages/landing/app/page.tsx

```diff
@@ -14,36 +14,37 @@ import { QAndAAccordion, TryStacklyBanner } from "@/components";
 export default function Home() {
   return (
     <main>
-      <section className="px-6 pt-20 pb-12 border-b border-gray-100">
-        <div className="max-w-6xl mx-auto">
-          <div className="space-y-6 text-center ">
-            <DisplayText>Stack crypto over time</DisplayText>
-            <HeadingText className="!font-medium text-em-med max-w-2xl mx-auto">
-              Stackly is a simple, non-custodial tool that does recurring buys
-              of a token at a choosen frequency. Making it easy to DCA.
-            </HeadingText>
-          </div>
-          <ButtonLink
-            target="_blank"
-            href={STACKLY_APP_URL}
-            size="lg"
-            width="fit"
-            className="!py-4 mx-auto text-lg !px-28 mt-7"
-          >
-            Launch app
-          </ButtonLink>
-          <div className="relative max-w-4xl mx-auto mt-16">
-            <Link passHref href={STACKLY_APP_URL}>
-              <Image
-                className="mx-auto border shadow-xl hover:shadow-2xl rounded-2xl border-surface-50"
-                alt="amount widget"
-                src="/assets/images/landing-amount-widget.png"
-                height={200}
-                width={512}
-              />
-            </Link>
-            <div className="absolute w-full -top-36 -z-10 h-[460px] bg-radial-gradient"></div>
-          </div>
+      <section className="px-6 pt-16 border-b border-gray-100 md:pt-20">
+        <div className="space-y-4 text-center md:space-y-6 ">
+          <DisplayText>DCA simplified</DisplayText>
+          <HeadingText className="!font-medium text-em-med max-w-2xl mx-auto">
+            Say goodbye to market timing and hello to a diversified portfolio
+            over time.
+          </HeadingText>
+        </div>
+        <ButtonLink
+          target="_blank"
+          href={STACKLY_APP_URL}
+          size="lg"
+          width="fit"
+          className="!py-4 mx-auto text-lg !px-16 md:!px-28 mt-8"
+        >
+          Stack now
+        </ButtonLink>
+        <div className="relative max-w-4xl mx-auto mt-12 mb-24 md:my-20">
+          <Link passHref href={STACKLY_APP_URL}>
+            {/* <div className="absolute w-0.5 h-4 bg-gray-600 bottom-[32px] left-2.5 animate-pulse"></div> */}
```

Comment:
```suggestion
```

Reply:
It wasn't suppose to commit that. Fixed!